src/InOneWeekend/camera.h

```diff
@@ -35,29 +35,11 @@ class camera {
     double aperture   = 0;
     double focus_dist = 10;
 
-    void initialize() {
-        auto theta = degrees_to_radians(vfov);
-        auto h = tan(theta/2);
-        auto viewport_height = 2.0 * h;
-        auto viewport_width = aspect_ratio * viewport_height;
-
-        w = unit_vector(lookfrom - lookat);
-        u = unit_vector(cross(vup, w));
-        v = cross(w, u);
-
-        origin = lookfrom;
-        horizontal = focus_dist * viewport_width * u;
-        vertical = focus_dist * viewport_height * v;
-        lower_left_corner = origin - horizontal/2 - vertical/2 - focus_dist*w;
-
-        lens_radius = aperture / 2;
-    }
-
     void render(const hittable& world) {
-        int image_height = static_cast<int>(image_width / aspect_ratio);
-
         initialize();
 
+        int image_height = static_cast<int>(image_width / aspect_ratio);
```

Comment:
This lline is above `initialize()` in the other two books. Not sure why you moved this, I'm fine with it above or below, but make it consistent

Reply:
Will do!